packages/apps/fortune/recording-oracle/package.json

```diff
@@ -8,6 +8,7 @@
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
+    "start:watch": "nest start --watch",
```

Comment:
Remove start:watch. We can just use `start:dev`, you can remove this NODE_ENV=development and call it as:
`NODE_ENV=development yarn start:dev`